src/polyfem/state/StateLoad.cpp

```diff
@@ -84,7 +84,8 @@ namespace polyfem
 		timer.start();
 		logger().info("Loading obstacles...");
 		obstacle = mesh::read_obstacle_geometry(
-			args["geometry"], args["boundary_conditions"]["obstacle_displacements"],
+			args["geometry"],
+			args["boundary_conditions"]["obstacle_displacements"], args["boundary_conditions"]["dirichlet_boundary"],
```

Comment:
So `obstacle_displacements` are still a field in the JSON? why not drop them and always use the `dirichlet_boundary`?

Reply:
it would require change all our jsons :)

Reply:
okay